File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.java
class Solution {
    public int countConsistentStrings(String allowed, String[] words) {
        int count = words.length;
        for(String word: words){
            for(int i=0;i<word.length();i++){
                if(allowed.indexOf(word.charAt(i)) == -1){
                    count--;
                    break;
                }
            }
        }
        return count;
    }
}

File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.js
/**
 * @param {string} allowed
 * @param {string[]} words
 * @return {number}
 */
var countConsistentStrings = function(allowed, words) {
    let count = words.length;
    for(let i=0; i< words.length; i++){
        for(let j=0; j< words[i].length; j++){
           if(!allowed.includes(words[i][j])){
                count--;
                break;
           }
        }
    }
    return count;
};